.travis.yml

```diff
@@ -22,7 +22,10 @@ before_install:
   - wget http://cs.sensiolabs.org/download/php-cs-fixer-v2.phar --output-document=php-cs-fixer.phar
 
 install:
-  - composer update --prefer-dist --no-interaction $COMPOSER_FLAGS
+  - |
+      composer global show hirak/prestissimo --quiet ||
+      travis_retry composer global require hirak/prestissimo --prefer-dist --no-interaction --no-progress
+  - composer update --prefer-dist --no-interaction --no-progress $COMPOSER_FLAGS
```

Comment:
This composer flags is suspicious, I can't find it used anywhere :P

Reply:
See 24cf39aa8f45364aa41f5681df508b7fe95fad9d

Reply:
so same as me ;)
if you want to drop it, let us not mix the things and drop it in separate PR please